channel-bot.py

```diff
@@ -36,11 +37,32 @@ def convert_channels_to_text(channels):
     return "".join(text_list).encode("utf-8")
 
 
+def read_config(config):
+    error_message = "Please create 'slack' section and contain options."
+    try:
+        config.read("config.ini")
+    except MissingSectionHeaderError:
+        print error_message
+        exit()
+
+    if not config.has_section("slack"):
+        print error_message
+        exit()
+
+
+def parse_required_option(config, option):
+    if not config.has_option("slack", option):
+        print "Please setting '{}' option in {} section.".format(option, "slack")
```

Comment:
[nits]
You assigned static value "slack" to second `{}` .
Is there reason you didn't write like `"Please setting '{}' option in slack section."` ?